eng/scripts/start-server.ps1

```diff
@@ -0,0 +1,66 @@
+param(
```

Comment:
I think a more descriptive name would be better here, e.g. `proxy-server.ps1` or something. Also calling it `start-server` makes it odd when you run `start-server.ps1 stop`